src/utils/gaa.js

```diff
@@ -251,11 +266,7 @@ export class GaaMeteringRegwall {
         .then(
           () =>
             // Only initialize Google Sign-In once.
-            self.gapi.auth2.getAuthInstance() ||
-            self.gapi.auth2.init({
-              'ux_mode': 'redirect',
-              'redirect_uri': redirectUri,
-            })
+            self.gapi.auth2.getAuthInstance() || self.gapi.auth2.init()
```

Comment:
From the dev docs, it seems that client_id is a required parameter to init. Are you using something else?

https://developers.google.com/identity/sign-in/web/reference#gapiauth2initparams

Reply:
I'm defining the Client ID in a meta tag. This is the simple method GSI tells developers about first in their docs https://developers.google.com/identity/sign-in/web/sign-in#specify_your_apps_client_id